src/libopensc/card-openpgp.c

```diff
@@ -98,6 +98,19 @@ static pgp_ec_curves_t	ec_curves_openpgp[] = {
 	{{{-1}}, 0} /* This entry must not be touched. */
 };
 
+/* v3.0+ supports: [RFC 4880 & 6637] 0x12 = ECDH, 0x13 = ECDSA */
```

Comment:
copy&paste comment which no longer reflects the reality.

Reply:
Thank you! And what is reality now? I thought that these RFCs are still valid